drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/DecisionTableXLSFileServlet.java

```diff
@@ -42,11 +47,19 @@
     @Inject
     private ExtendedDecisionTableXLSService decisionTableXLSService;
 
+    private SessionProvider sessionProvider;
+
+    @Override
+    public void init( final ServletConfig config ) throws ServletException {
+        super.init( config );
+        sessionProvider = ServletBootstrapUtil.getService( config ).getSessionProvider();
```

Comment:
Wouldn't it be better to do this in init of AbstractFileServlet and make it a protected field there?